packs/data/feat-effects.db/effect-harrow-the-fiend.json

```diff
@@ -0,0 +1,47 @@
+{
+    "_id": "MSkspeBsbXm6LQ19",
+    "img": "systems/pf2e/icons/equipment/cursed-items/euryale-curse-card.webp",
+    "name": "Effect: Harrow the Fiend",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>You gain a +1 status bonus on all Strikes or spell attack rolls against fiends.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "encounter",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "target:trait:fiend"
+                ],
+                "selector": "attack",
```

Comment:
This should probably be:
```
"selector": [
	"spell-attack-roll",
	"strike-attack-roll"
]
```

Reply:
No, just `attack` covers both, see inspire courage